src/snowflake/snowpark/table.py

```diff
@@ -470,28 +452,6 @@ def update(
         )
         return _get_update_result(result) if block else result
 
-    @overload
-    def delete(
-        self,
-        condition: Optional[Column] = None,
-        source: Optional[DataFrame] = None,
-        *,
-        statement_params: Optional[Dict[str, str]] = None,
-        block: bool = True,
-    ) -> DeleteResult:
-        ...  # pragma: no cover
```

Comment:
Is there anything we have to do to note that the docs are going to change?

Reply:
Nope, the signature will be extracted from the code as long as we have the entry `~Table.delete` in `docs/source/table.rst`.